tox.ini

```diff
@@ -44,6 +44,8 @@ commands = /bin/bash -c '! sphinx-build -b linkcheck docs {envtmpdir}/linkcheck
 
 [testenv:int]
 deps = {[_test]deps}
-setenv = INTEGRATION_TEST = 1
+setenv =
+    INTEGRATION_TEST = 1
+    GENIUS_ACCESS_TOKEN = <your genius access token>
```

Comment:
Hmm; I'm not sure we should be recommending that people put their access token here in this file. It seems to run the risk of people accidentally committing the token here (and in any case, it's sort of inconvenient to avoid committing temporary changes to files like this). Maybe we should just recommend that people do it using their shell, and include it in the `passenv` list below?